[2024-04-01T21:30:02.282+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_get_tmd_api.get_data_api scheduled__2024-04-01T18:30:00+00:00 [queued]>
[2024-04-01T21:30:02.311+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_get_tmd_api.get_data_api scheduled__2024-04-01T18:30:00+00:00 [queued]>
[2024-04-01T21:30:02.313+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-01T21:30:02.358+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_data_api> on 2024-04-01 18:30:00+00:00
[2024-04-01T21:30:02.370+0000] {standard_task_runner.py:60} INFO - Started process 187509 to run task
[2024-04-01T21:30:02.381+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'test_get_tmd_api', 'get_data_api', 'scheduled__2024-04-01T18:30:00+00:00', '--job-id', '226', '--raw', '--subdir', 'DAGS_FOLDER/test_get_tmd_api.py', '--cfg-path', '/tmp/tmpma6p7uc6']
[2024-04-01T21:30:02.387+0000] {standard_task_runner.py:88} INFO - Job 226: Subtask get_data_api
[2024-04-01T21:30:02.521+0000] {task_command.py:423} INFO - Running <TaskInstance: test_get_tmd_api.get_data_api scheduled__2024-04-01T18:30:00+00:00 [running]> on host c5a6ac367c49
[2024-04-01T21:30:02.770+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_get_tmd_api' AIRFLOW_CTX_TASK_ID='get_data_api' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T18:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T18:30:00+00:00'
[2024-04-01T21:30:20.935+0000] {test_get_tmd_api.py:43} INFO - **** filename : data_2024_04_02_0400.json
[2024-04-01T21:30:20.948+0000] {test_get_tmd_api.py:49} INFO - JSON data saved to /opt/***/dags/data_get.json
[2024-04-01T21:30:20.949+0000] {test_get_tmd_api.py:53} INFO - local file path : /opt/***/dags/data_get.json
[2024-04-01T21:30:20.950+0000] {test_get_tmd_api.py:57} INFO - bucket name : ds525-capstone-test-49
[2024-04-01T21:30:20.950+0000] {test_get_tmd_api.py:60} INFO - dst : data_test/data_2024_04_02_0400.json
[2024-04-01T21:30:20.974+0000] {connection.py:269} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-04-01T21:30:20.986+0000] {base.py:83} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-04-01T21:30:21.946+0000] {gcs.py:557} INFO - File /opt/***/dags/data_get.json uploaded to data_test/data_2024_04_02_0400.json in ds525-capstone-test-49 bucket
[2024-04-01T21:30:21.952+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-01T21:30:21.986+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=test_get_tmd_api, task_id=get_data_api, execution_date=20240401T183000, start_date=20240401T213002, end_date=20240401T213021
[2024-04-01T21:30:22.037+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-01T21:30:22.078+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
